doc_source/tls.md

```diff
@@ -22,7 +22,9 @@ For additional requirements, select the issuer of the certificate that you're us
 
 ### ACM PCA<a name="certificate-pca"></a>
 
-The certificate must be stored in ACM in the same Region and IAM user, as the mesh endpoint that will use the certificate\. If you don't have an ACM Private CA, then you must [create one](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaCreateCa.html) before you can request a certificate from it\. For more information about requesting a certificate from an existing ACM PCA using ACM, see [Request a Private Certificate](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-private.html)\. The certificate cannot be a public certificate\.
+The certificate must be stored in ACM in the same Region and AWS Account, as the virtual node that will use the certificate\. If you don't have an ACM Private CA, then you must [create one](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaCreateCa.html) before you can request a certificate from it\. For more information about requesting a certificate from an existing ACM PCA using ACM, see [Request a Private Certificate](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-private.html)\. The certificate cannot be a public certificate\.
```

Comment:
Can you rename `virtual node` to `mesh endpoint` as this is applicable to both Virtual Node and Virtual Gateway?

Reply:
Bah, I keep neglecting this, good call. I didn't properly read the definition of endpoint above.